content/about/index.md

```diff
@@ -10,4 +10,32 @@ Packit provides tooling and automation to integrate upstream open source project
 
 https://github.com/packit-service/packit
 
+
+### The key principles of packit
+
+* Reuse existing tools and services: don't reinvent the wheel.
+
+* You can consume packit in two forms:
+  * Packit tool can run on your laptop and you run commands you want.
+  * Packit service reacts to events and performs actions which you
+    defined in your `packit.yaml`.
+
+* We don't break backwards compatibility just like that.
+
+* Packit has a deprecation policy:
+  * We can mark a behaviour as deprecated.
+  * Deprecated content can be removed or changed after at least 3 minor
+    releases.
+  * Deprecated content is advertised in our
+    [changelog](https://github.com/packit-service/packit/blob/master/CHANGELOG.md),
+    printed in terminal or tracked in your issue tracker.
+
+* We care about artifacts which Fedora supports: at the moment it's RPMs,
+  modules and container images.
+  * Our initial focus is solely on RPMs.
+
+* Packit respects Fedora guidelines.
+
+* We want latest content in Fedora Rawhide, but only if it works.
```

Comment:
The sentence `only if it works` has a general meaning. 
You can think, content can be installed, content can be executed. Does it cover the tests?
I don't think, general meaning is correct. Can you please rephrase it a bit more?

Reply:
please suggest

Reply:
My suggestions:

`...., but only if it works, as CI tests passed, package build finishes with success by koji.`